go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -1518,17 +1542,20 @@ func commonFlows(subnets []*net.IPNet, bridge *bridgeConfiguration) ([]string, e
 	// Due to the fact that ovn-controllers on different nodes apply the changes independently,
 	// there is a chance that the pod traffic will reach the egress node before it configures the SNAT flows.
 	// Drop pod traffic that is not SNATed, excluding local pods(required for ICNIv2)
+	defaultNetConfig := bridge.netConfig[types.DefaultNetworkName]
 	if config.OVNKubernetesFeature.EnableEgressIP {
 		for _, clusterEntry := range config.Default.ClusterSubnets {
 			cidr := clusterEntry.CIDR
 			ipPrefix := "ip"
 			if utilnet.IsIPv6CIDR(cidr) {
 				ipPrefix = "ipv6"
 			}
-			// table 0, drop packets coming from pods headed externally that were not SNATed.
-			dftFlows = append(dftFlows,
-				fmt.Sprintf("cookie=%s, priority=104, in_port=%s, %s, %s_src=%s, actions=drop",
-					defaultOpenFlowCookie, ofPortPatch, ipPrefix, ipPrefix, cidr))
+			for _, netConfig := range bridge.netConfig {
+				// table 0, drop packets coming from pods headed externally that were not SNATed.
+				dftFlows = append(dftFlows,
+					fmt.Sprintf("cookie=%s, priority=104, in_port=%s, %s, %s_src=%s, actions=drop",
+						defaultOpenFlowCookie, netConfig.ofPortPatch, ipPrefix, ipPrefix, cidr))
```

Comment:
This will happen in @martinkennelly 's PR so I have moved this flow back to only being for default network currently. @martinkennelly can take care of making it UDN aware in EIP PR.

Reply:
great catch tim!

I have moved this back to being:
```
for _, clusterEntry := range config.Default.ClusterSubnets {
			cidr := clusterEntry.CIDR
			ipPrefix := "ip"
			if utilnet.IsIPv6CIDR(cidr) {
				ipPrefix = "ipv6"
			}
			// table 0, drop packets coming from pods headed externally that were not SNATed.
			dftFlows = append(dftFlows,
				fmt.Sprintf("cookie=%s, priority=104, in_port=%s, %s, %s_src=%s, actions=drop",
					defaultOpenFlowCookie, defaultNetConfig.ofPortPatch, ipPrefix, ipPrefix, cidr))
		}
```